Client/mods/deathmatch/logic/CPedSync.cpp

```diff
@@ -67,12 +78,17 @@ void CPedSync::DoPulse()
 void CPedSync::AddPed(CClientPed* pPed)
 {
     m_List.push_front(pPed);
+    pPed->SetSyncing(true);
 }
 
 void CPedSync::RemovePed(CClientPed* pPed)
 {
     if (!m_List.empty())
         m_List.remove(pPed);
+
+    pPed->SetSyncing(false);
+    // Reset vehicle in/out stuff in case the ped was entering/exiting
+    pPed->ResetVehicleInOut();
```

Comment:
- [ ] @Zangomangu  Eh so I know SetSyncing(false) is only called in one  place (here) but I think the point of having function sinstead of directly accessing members is so that we can do these sort of side effects if we need to. Should `ResetVehicleInOut()` actually just be inside the SetSyncing(false) implementation?

Reply:
Good idea, moved it into SetSyncing